README.md

```diff
@@ -23,25 +44,83 @@ The #1 best way to understand and address the weaknesses in an API's design and
 
 Whenever feasible, design an API in parallel with an accompanying integration of that API.
 
+A few methods to accomplish this include:
+* Identifying an internal GSA organization to use your API while also publishing it publicly.
+* Creating a web page with a search feature that uses the API.
+* Modifying existing web pages or web applications to use the API instead of direct access to the database.
+
+## Developers Are Your End Users
+Consider developers who will be using your APIs. Their path to using your API will include discovery and inital investigation, sample API calls, development and testing, deployment and production usage. Consider each of these functions in your documentation, support, and change notification process. Consider performing formal [API Usability Testing](### Default to REST
+These standards assuming the APIs will be generally "Restful". However, recommendations are also provided for SOAP web services.
+) to understand the developer experience in using your API.
+
+### Add Your API To The GSA API Directory
+A directory of GSA public APIs is available at [open.gsa.gov/developer](open.gsa.gov/developer/). You can add your API to this directory by posting an issue or pull request in the [GitHub repository](https://github.com/GSA/open.gsa.gov).
```

Comment:
From Cindy S: The GSA API Directory’ open.gsa.gov/developer returns a 404 Error this is not the web page you are looking for.